src/OpenFeature/OpenFeature.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
```

Comment:
I thought `net462` was included in `netstandard2.0` https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0 ?